packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -148,6 +166,7 @@ export default class SingleSelect extends React.Component<Props, State> {
         this.setState({
             open: opened,
             keyboard,
+            searchText: "",
```

Comment:
I wonder if there's any situations where we'd want to maintain the search text. 🤔 No action req'd.